packages/auth/android/src/main/java/io/invertase/firebase/auth/ReactNativeFirebaseAuthModule.java

```diff
@@ -941,7 +949,13 @@ public void confirmationResultConfirm(
   ) {
     FirebaseApp firebaseApp = FirebaseApp.getInstance(appName);
     FirebaseAuth firebaseAuth = FirebaseAuth.getInstance(firebaseApp);
-
+    if(mVerificationId == null){
+      String verificationId = sharedPreferences.getString("verificationId","null");
+      mVerificationId = verificationId;
+      Log.d(TAG,
+              "_confirmVerificationCode: Empty verificationId , using local verificationId:" + verificationId
+      );
```

Comment:
```suggestion
```suggestion
      Log.d(TAG, "_confirmVerificationCode: Empty verificationId, using local value: " + verificationId);
``````